benchmarks/gups/gups.cpp

```diff
@@ -122,8 +122,9 @@ int run_benchmark(const Index indicesCount, const Index dataCount,
   DataView data("data", dataCount);
   Kokkos::parallel_for(
       "init-data",
-      Kokkos::RangePolicy<Kokkos::DefaultExecutionSpace>(0, dataCount),
-      KOKKOS_LAMBDA(const int i) { data[i] = 10101010101; });
+      Kokkos::RangePolicy<Kokkos::DefaultExecutionSpace,
+                          Kokkos::IndexType<Index>>(0, dataCount),
+      KOKKOS_LAMBDA(const Index i) { data[i] = 10101010101; });
```

Comment:
hmm ... what impact does that have on the performance of different gpus?

Reply:
The point is that it's now properly customizable.

Reply:
That is a good step, sure. But why also change it to `int64_t` without looking at the implications?